exercises/crypto-square/.meta/generator/crypto_square_case.rb

```diff
@@ -0,0 +1,46 @@
+require 'generator/exercise_case'
+
+class CryptoSquareCase < Generator::ExerciseCase
+
+  # TODO: remove guard clause when #encoded method
+  # is added to crypto_square.rb
```

Comment:
So the canonical data is a bit off for this exercise. There are several tests in the canonical data that are expecting a method named ```encode``` or possibly ```encoded``` to be defined on the ```Crypto``` class. However, this method is missing from ```crypto_square.rb```